packages/web-components/src/components/rux-monitoring-progress-icon/rux-monitoring-progress-icon.tsx

```diff
@@ -136,6 +137,10 @@ export class RuxMonitoringProgressIcon {
         }
     }
 
+    componentDidRender() {
+        this.handleNotificatonWidth()
```

Comment:
one issue with this approach (and also in monitoring icon maybe), is that the width doesn't get smaller if the notifications get lower. it will correctly expand if you go from 99 -> 999, but if you go from 999 -> 99, it still has the maximum width. this means that when the notification count changes, it'll cause the layout to jank around as the widths adjust. I think that part is unavoidable unfortunately, but we should at least look into having it update when you go from 999 -> 99.